src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -88,6 +101,8 @@ public String getConfirmation(@PathVariable String companyNumber,
         try {
             PayableLateFilingPenalty payablePenalty = payablePenaltyService
                     .getPayableLateFilingPenalty(companyNumber, payableRef);
+            lateFilingPenalties = penaltyPaymentService.getLateFilingPenalties(companyNumber, penaltyRef);
```

Comment:
This opens up question on whether we need to make this generic / not lfp specific or what we'd do in that instance. Need to have a think for future task around changing / refactoring this, but tbf that can come later.